screens/Home.js

```diff
@@ -98,10 +112,16 @@ class HomeScreen extends React.Component {
   }
   _onRefresh = () => {
     this.setState({ refreshing: true });
-    this.fetchUserWorkouts((workouts) => {
+    this.fetchUserWorkouts(workouts => {
       this.setState({ refreshing: false, workouts: workouts });
     });
+  };
+
+  deleteWorkout(workout) {
+    const user = firebase.auth().currentUser;
```

Comment:
Would be nice to ask for a confirmation before deleting a workout, in case if someone clicks on it accidentally. Similar to when closing create workout window.

Reply:
Well, the matter is that you already need to long press the workout, so you're unlikely to delete it accidentally. Not a big fan of too many clicks. What do you think?

Reply:
What if someone clicks delete when they actually wanted to click share or other many option.
Btw you should always confirm a deletion in every application, it's just a well known practice.

Reply:
@AlexeiGitH  Done 
<img width="363" alt="Screen Shot 2019-07-14 at 7 51 12 PM" src="https://user-images.githubusercontent.com/26641473/61190841-d4e77f80-a670-11e9-8c53-fc39fd85d806.png">